substrate/client/transaction-pool/src/fork_aware_txpool/view_store.rs

```diff
@@ -685,7 +684,7 @@ where
 		// be in the pool.
```

Comment:
Doesn't removing from view mean removing from the pool too?

Reply:
No, the pool here means the highest level pool (fork-aware) which may maintain a number of views (one active view per fork). Thus removing from the single view does not mean removing from the pool - tx can still be present in other views.